### DISCUSSION TITLE
No Firebase Support?

### DISCUSSION THREAD

**Discussion by davidjeba** [Budibase/budibase#4792] [Help] [answered]
https://budibase.com/integration/

Can you provide Flutter SDK with Firebase support {Google ecosystem}

as Flutter has cross platform app building capabilities
it would be great if budibase can use firebase ecosystem for SaaS development
same firebase database can be used by flutter based mobile apps without API middleware

[admin + website + mobile apps} all built/connected on same data

**Comment by melohagan** [comment, marked as answer]
Hi @davidjeba 

Firebase integration is currently on the roadmap: https://github.com/Budibase/budibase/projects/10

In the meantime, you can use the REST API connector to communicate with your firebase instance:

 - https://firebase.google.com/docs/reference/rest/database
 - https://docs.budibase.com/docs/rest

**Reply by melohagan** [reply]
@davidjeba The Firestore data connector is now available: https://docs.budibase.com/docs/firestore
